static/src/javascripts/projects/common/utils/closest.js

```diff
@@ -0,0 +1,18 @@
+define(function () {
+    /**
+     * Adds a wrapper around Element.closest()
+     */
+    var matches = 'matches' in Element.prototype ? 'matches' : 'msMatchesSelector';
```

Comment:
are we sure that all polyfilled browsers have either `matches` or `msMatchesSelector` in `Element` proto?